examples/image-to-text/run_pipeline.py

```diff
@@ -143,7 +154,8 @@ def main():
     )
     parser.add_argument(
         "--use_kv_cache",
-        action="store_true",
+        type=str2bool,
+        default=True,
```

Comment:
```suggestion
        action="store_true",
        default=None,
```
this should work, don't need to set to True by default

Reply:
This cannot solve the problem, as if we set default to None, it will pass down this param to modeling part, and change the value of `use_cache` in `generation_config` here: [L671](https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/generation/utils.py#L671)